src/box/iproto.cc

```diff
@@ -1958,6 +2033,9 @@ tx_accept_msg(struct cmsg *m)
 	tx_fiber_init(msg->connection->session, msg->header.sync);
 	tx_prepare_transaction_for_request(msg);
 	msg->connection->iproto_thread->tx.requests_in_progress++;
+	rlist_add_entry(&msg->connection->tx.inprogress, msg,
+			in_inprogress);
+	msg->fiber = fiber();
```

Comment:
Now, we could probably drop the `accepted` flag and use `msg->fiber != NULL` instead.

Reply:
Done.

<details>
<summary>diff</summary>

```diff
--- a/src/box/iproto.cc
+++ b/src/box/iproto.cc
@@ -451,12 +451,6 @@ struct iproto_msg
        struct stailq_entry in_stream;
        /** Stream that owns this message, or NULL. */
        struct iproto_stream *stream;
-       /**
-        * True if message processing in tx thread is started. The flag is
-        * used to prevent double accounting of message in case of iproto
-        * override fallback to original handler.
-        */
-       bool accepted;
        /** Link in connection->tx.inprogress. */
        struct rlist in_inprogress;
        /** TX thread fiber that processing this message. */
@@ -848,7 +842,7 @@ iproto_msg_new(struct iproto_connection *con)
        msg->close_connection = false;
        msg->connection = con;
        msg->stream = NULL;
-       msg->accepted = false;
+       msg->fiber = NULL;
        rmean_collect(con->iproto_thread->rmean, IPROTO_REQUESTS, 1);
        return msg;
 }
@@ -2023,9 +2017,8 @@ static inline struct iproto_msg *
 tx_accept_msg(struct cmsg *m)
 {
        struct iproto_msg *msg = (struct iproto_msg *) m;
-       if (msg->accepted)
+       if (msg->fiber != NULL)
                return msg;
-       msg->accepted = true;
        tx_accept_wpos(msg->connection, &msg->wpos);
        tx_fiber_init(msg->connection->session, msg->header.sync);
        tx_prepare_transaction_for_request(msg);
```

</details>